app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -329,8 +327,13 @@ class ThreadFragment : Fragment() {
                 return@observe
             }
 
-            fetchMessagesHeavyData(messages)
             threadAdapter.submitList(messages)
+
+            threadViewModel.fetchMessagesHeavyData(messages)
```

Comment:
I placed them together to regroup both fetch methods. And placing them after the submitList rather than before is is to make it clearer that these functions are not doing something logic to prepare the list before submitting it